CliFx.Tests/Services/HelpTextRendererTests.cs

```diff
@@ -106,6 +106,23 @@ private static IEnumerable<TestCaseData> GetTestCases_RenderHelpText()
                     "-h|--help", "Shows help text."
                 }
             );
+
+            yield return new TestCaseData(
+                CreateHelpTextSource(
+                    new[] { typeof(SampleValueOptionCommand) },
+                    typeof(SampleValueOptionCommand)),
+
+                new[]
+                {
+                    "Description",
+                    "SampleValueOptionCommand description.",
+                    "Usage",
+                    "testapp sampleval", "-f option_f_value", "[options]",
```

Comment:
Probably would make sense to add tests for at least one command that doesn't use `SampleValue`.

Reply:
Once removed the `SampleValue` property this is not needed anymore.